scripting/pugsetup.sp

```diff
@@ -18,6 +19,7 @@
 #define ALIAS_LENGTH 64
 #define COMMAND_LENGTH 64
 #define LIVE_TIMER_INTERVAL 0.3
+#define LoopClients(%1)    for (int %1 = 1; %1 <= MaxClients; %1++)
```

Comment:
Don't use macros, write full code for better readability

Reply:
idk what do you mean, this already make better readability.